tea_base/__init__.py

```
@@ -0,0 +1,582 @@
+        """Set product version."""
+        self.product["version"] = version
+        return True
+
+    def get_version(self):
+        """Return product version."""
+        return self.product["version"]
+
+    def add_leaf(self, leaf):
+        """Add leaf to product."""
+        from product import leaf
+        if not isinstance(art, leaf):
```

Ruff

Undefined name `art`

(F821)